pyplot/baselineFits/SvtBlFitSummaryPlots.py

```diff
@@ -0,0 +1,144 @@
+import ROOT as r
```

Comment:
Please consider switching to "argparse" instead of "optparse", and switch from "%s_blabla"%(strvar) to "{s}_blabls".format(strvar). Both of which are Python2 and Python3 compatible and have long term support.